internal/signature/testdata/key

```diff
@@ -0,0 +1,82 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
Note for the reviewer: this key is technically unused, however it is useful to have it if in the future we want to add some other test that are signed by the same entity as the other tests.

Obviously, like the filename says, this is a test key and its sole purpose is to run these tests.